aten/src/ATen/native/quantized/cpu/fake_quantize_per_tensor_affine.cpp

```diff
@@ -131,16 +127,5 @@ class FakeQuantizePerTensorAffineOp_backward : public c10::OperatorKernel {
     at::Tensor mask = mask_min * mask_max;
     dX = mask.type_as(dY) * dY;
```

Comment:
btw, I'm not an expert in QAT, but just curious - why does zeroing out like this make sense?

also, if it ever becomes a bottleneck - we should fuse these ops into one loop instead of tensor ops

Reply:
This is just a naive implementation, we can have better gradient later.

for bottleneck, the op we'll use is actually the cuda implementation which is slightly different than this one.

Reply:
@dzhulgakov: We zero out the gradients when the output is saturated, here the output is not a function of the input, so the contribution to the gradient is zero.

Reply:
We need to do rounding after addition of zero_point:
```
at::Tensor Xq = (X * inv_scale + 0.5 + zero_point).floor();
```